### PR TITLE
Implement rmt-products list command

### PR DESCRIPTION
**Pull request by funglaub**
Not sure about requiring 'erb'...

### CODE REVIEW

**Comment by ikapelyukhin** [comment]
@funglaub It doesn't look like you are rendering any templates in this PR, I don't see why `erb` would be required.
Also some Rubocop checks are failing, but those should be fixable using Rubocop's auto-correction (`rubocop -a`).

**Comment by funglaub** [comment]
Sorry about that, I will have a look at that in the evening :)

**Comment by ikapelyukhin** [comment]
@funglaub Sure, no problem. :+1:

File: lib/database.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 require 'active_record'
+require 'erb'
```
Comment (ikapelyukhin) [inline lib/database.rb]: OK, I've taken another look -- requiring `erb` here is actually fine, there's some rendering happening in database.yml.
(In other `bin` scripts `erb` gets pulled in by `rmt/config`)

**Pull request closed without merging**